### PR TITLE
Swap links

### PR DESCRIPTION
**Pull request by hrbrnd**
Added Hyperlinks to previously known Column Website. Modified Looping block from date and also removed the addition of the hyperlink it had. 

Changed column headers to more usable names.

### CODE REVIEW

File: app/views/contracts/index.html.erb [hunk 1]
```diff
@@ -28,15 +28,28 @@
...
+                </ul>
+              </td>
               <td><%= contract.application %></td>
               <% if contract.contract_revisions.size > 0 %>
               <td>
                 <ul class="contract-revision-list">
                 <% contract.contract_revisions.each do |revision| %>
                   <li>
                     <% if revision.tos_url.present? %>
-                      <a href="http://<%= revision.tos_url %>" target="_blank">
```
Comment (chen-robert) [inline app/views/contracts/index.html.erb]: Why was this line removed?

Comment (hrbrnd) [reply app/views/contracts/index.html.erb]: Because that made the date a hyperlink which we were switching to the previously known website column(now it is the Link to Terms column)

Comment (chen-robert) [reply app/views/contracts/index.html.erb]: Okay that makes sense.

**Review by chen-robert — DISMISSED** [review]

File: app/views/contracts/index.html.erb (hunk 1 above)
Comment (monroemann) [reply app/views/contracts/index.html.erb]: @chen-robert Test it out and see if it works.  I got him started with it.

File: app/views/contracts/index.html.erb [hunk 2]
```diff
@@ -28,15 +28,28 @@
...
                         <%= revision.contract_date.strftime('%m/%d/%Y') %>
                       </a>
                     <% else %>
```
Comment (monroemann) [inline app/views/contracts/index.html.erb]: However, this if statement seems to do nothing now.  Either way, it seems the same data is printed out.  Am I correct?  I'm just looking at this quickly

Comment (hrbrnd) [reply app/views/contracts/index.html.erb]: The IF ELSE can be swapped to just an IF statement to clean the code up. The IF is still making sure that there is actually a URL assigned to go with it more like precautionary so I feel like it is a good check to still have.

Comment (taratip) [reply app/views/contracts/index.html.erb]: I think it's okay to remove IF ELSE. It shows contract_date in both cases and we don't need to check URL since we will not render the link for that URL.

**Review by emestabillo — DISMISSED** [review]

File: app/views/contracts/index.html.erb (hunk 2 above)
Comment (hrbrnd) [reply app/views/contracts/index.html.erb]: You guys are correct, I made the changes to clean it up

**Review by taratip — APPROVED** [review]

**Review by emestabillo — APPROVED** [review]
